deepcell/utils/io_utils.py

```diff
@@ -159,8 +164,12 @@ def get_images_from_directory(data_location, channel_names):
         for j in range(n_channels):
             img_path = os.path.join(data_location, img_list_channels[j][stack_iteration])
             channel_img = get_image(img_path)
-            if data_format == 'channels_first':
+            if (data_format == 'channels_first') and (channel_img.ndim > 2):
+                all_channels[0, j, :, :, :] = channel_img
+            elif data_format == 'channels_first':
                 all_channels[0, j, :, :] = channel_img
+            elif channel_img.ndim > 2:
+                all_channels[0, :, :, :, j] = channel_img
```

Comment:
I think this can be simplified by not enumerating each dimension:

```python
all_channels[0, j, ...] = channel_img

# or

all_channels[0, ..., j] = channel_img
```

Reply:
yeah, done